engine-tests/src/main/java/org/terasology/engine/testUtil/Assertions.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.engine.testUtil;
+
+import com.google.common.collect.Iterables;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class Assertions {
+    @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+    public static void assertNotEmpty(Optional<?> obj) {
+        if (!obj.isPresent()) {
+            fail("Empty result.");
+        }
+    }
+
+    public static void assertNotEmpty(Iterable<?> obj) {
+        if (Iterables.isEmpty(obj)) {
+            fail("Empty result.");
+        }
+    }
```

Comment:
I'm surprised this is not available by default 🧐 Are these now available for module tests as well?

Reply:
Right? There are things for Arrays and Lists and other Collections, but I didn't see anything in junit Assertions for Iterable.

Reply:
oh, and yes, public classes in `engine-tests` are available to module tests, I believe.